### PR TITLE
Add Service Account Policy restriction improvement

### PR DESCRIPTION
**Pull request by jinapurapu**
Screenshot from 2022-04-28 12-59-21

closes https://github.com/miniohq/engineering/issues/809

### CODE REVIEW

**Review by adfost — COMMENTED** [review]
Why are these spaces being deleted in swagger?

**Comment by cniackz** [comment]
Tested locally, I can see the `policy` being populated:

Screen Shot 2022-05-02 at 8 03 08 PM

**Review by cniackz — DISMISSED** [review]
LGTM

**Review by dvaldivia — APPROVED** [review]

File: restapi/client-admin.go [hunk 1]
```diff
@@ -201,6 +202,8 @@ func (ac AdminClient) listPolicies(ctx context.Context) (map[string]*iampolicy.P
 // implements madmin.ListCannedPolicies()
 func (ac AdminClient) getPolicy(ctx context.Context, name string) (*iampolicy.Policy, error) {
 	praw, err := ac.Client.InfoCannedPolicy(ctx, name)
+	tempPolicy, _ := iampolicy.ParseConfig(bytes.NewReader(praw))
+	fmt.Println("client.getPolicy - tempPolicy.statements:", tempPolicy.Statements)
```
Comment (prakashsvmx) [inline restapi/client-admin.go:206]: is this added for debugging purposes?

**Review by prakashsvmx — DISMISSED** [review]
LGTM :+1:

**Review by dvaldivia — APPROVED** [review]

**Review by kaankabalak — APPROVED** [review]

**Pull request merged**